### ISSUE TITLE
Passing touches from Invisible VC to Camera/Root View (Discussion)

### ISSUE THREAD

**Issue by otymartin** [uias/Pageboy-Layouts#1]
This is more of a discussion as I've been trying to solve this problem unique to this layout.
Based on the view hierarchy you recommended
```swift
-> ParentViewController (CameraViewController)
     -> PageboyViewController (ScrollView found in UIPageViewController)
          -> LeftViewController
          -> InvisibleViewController
          -> RightViewController
```
I can get touches from `InvisibleViewController` down to `PageboyViewController` by just setting `InvisibleViewController view alpha to zero` or subclassing the UIView & performing a hitTest and returning `nil` if `hitTestView = self`

Beyond that, I a hitTest needs to be performed to pass touches through the `ScrollView` in the `UIPageViewController` found in Pageboy however i think its private and can't be subclassed which is where the road ends.

**Comment by msaps** [comment]
Hi @otymartin, I haven't had much time to take a look at this properly of late. I'll try and take a look at some point this week and see if I can figure something out

**Comment by msaps** [comment]
Please do update if you make any progress though!

**Comment by otymartin** [comment]
@msaps 
Hey I pretty much hit a wall with this. The only thought I have is to drop `UIPageViewController` and maybe adopt something like [`EMPageViewController`](https://github.com/emalyak/EMPageViewController)  so that the scrollview can be subclassed to pass touch events below. Doing that is beyond my technical capacity though🙈 I'm playing around with it at the moment.

**Comment by otymartin** [comment]
I've totally failed at this..more because I cant wrap my head around how hitTesting works and this layout has so many subviews its hard to follow the logic :(

**Comment by msaps** [comment]
@otymartin really am sorry about not getting back to you on this, I will try my best to take another look at it this week! 😄

**Comment by otymartin** [comment]
@msaps heh I can tell your busy. But yea if you can handle this that would be super awesome🔥🔥
